wagtail/admin/api/actions/create_alias.py

```diff
@@ -0,0 +1,52 @@
+from django.core.exceptions import ValidationError as DjangoValidationError
+from django.shortcuts import get_object_or_404
+from rest_framework import fields, status
+from rest_framework.exceptions import ValidationError
+from rest_framework.response import Response
+from rest_framework.serializers import Serializer
+
+from wagtail.api.v2.utils import BadRequestError
+from wagtail.core.actions.create_alias import CreatePageAliasAction, CreatePageAliasIntegrityError
+from wagtail.core.models import Page
+
+from .base import APIAction
+
+
+class CreatePageAliasAPIActionSerializer(Serializer):
+    parent_page_id = fields.IntegerField(required=False)
```

Comment:
I think for consistency with copy, we should rename this to ``destination_page_id``: https://github.com/wagtail/wagtail/blob/main/wagtail/admin/api/actions/copy.py#L18